scripts/cmake/rustlang.cmake

```diff
@@ -85,6 +85,21 @@ function(build_rust_archives)
         list(APPEND RUST_BUILD_CARGO_ENV MACOSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET})
     endif()
 
+    if(ANDROID)
+        if(CMAKE_HOST_SYSTEM_NAME STREQUAL Linux)
+            set(ANDROID_HOST_TAG linux-x86_64)
+        elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL Darwin)
+            set(ANDROID_HOST_TAG darwin-x86_64)
+        elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
+            set(ANDROID_HOST_TAG windows-x86_64)
+        endif()  
```

Comment:
What if the system host name is not any of these 3, what happens in that case?

Reply:
CMAKE_HOST_SYSTEM_NAME - is the "what os are you currently running cmake on?" 

If you have something that is not listed in here, you are running on a pc that should not have access to the android-ndk anyway. 😅 

https://developer.android.com/ndk/guides/other_build_systems#overview

Reply:
Makes sense, should there be default `else` statement that errors out just in case?

Reply:
Added :)